cli.js

```diff
@@ -96,5 +97,18 @@ function handleFkillError(processes) {
 if (cli.input.length === 0) {
 	init(cli.flags);
 } else {
-	fkill(cli.input, cli.flags).catch(() => handleFkillError(cli.input));
+	let pids = [];
+	const promises = [];
+	for (let i = 0; i < cli.input.length; i++) {
+		if (cli.input[i][0] === ':') {
+			promises.push(portToProc(parseInt(cli.input[i].slice(1), 0)));
+		} else {
+			pids.push(cli.input[i]);
+		}
+	}
+	Promise.all(promises).then(promisedPids => {
+		promisedPids = promisedPids.filter(pid => pid);
```

Comment:
Isn't rejecting a Promise like throwing an error? I'm not sure it's an error if you ask for the process using port X and there is no process using that port. That feels to me like it should return undefined.

If we want to treat undefined in this use case as an error, we can do - though I think i'd be more in favour or silently ignoring the command. At least if there's positive hits in the request too?

Am I making sense?